runtime/testruntime/testruntime.go

```diff
@@ -244,7 +260,7 @@ func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string) {
 			{
 				Type:   "druid",
 				Name:   "druid",
-				Config: map[string]string{"dsn": "http://localhost:8888/druid/v2/sql"},
+				Config: map[string]string{"dsn": fmt.Sprintf("https://%s:%s@druid.metrics.in.rilldata.io/druid/v2/sql", splits[0], splits[1])},
```

Comment:
Remove – we should use test containers instead of running against a prod cluster in testing. If we really need to run against an external cluster, it should be a dedicated cluster and generically configured

Reply:
I think tests are better than no tests

Reply:
`and it looks bad to open source contributors who don't get a chance to run or modify these tests` I will look even worse if get production Druid-based errors
`This both has the potential to overload a production cluster we rely on for more important use cases, makes it more difficult for folks internally to configure the tests` @nishantmonu51 I wonder if we should start a dedicated cluster for CI tests?

Reply:
There's a dedicated container implemented but it was switched off for performance reason. 
"Regarding this PR, I think bigger changes to Druid testing improvements should be done in a dedicated PR."
I expected it will be an easy addition but it turns we want to postpone Druid tests again

Reply:
Does changing `metrics-in` to `master-in` cluster resolve the issue?